src/app/pages/News/News.tsx

```diff
@@ -1,98 +1,112 @@
-    'https://fortnite-api.com/v2/news/stw'
-  );
-  const saveTheWorldNews = saveTheWorldData?.data.messages;
+  const { data: creativeData, isLoading: creativeLoading } =
+    useFetch<newsFromAPI>('/api/news/?mode=creative');
+  const creativeNews = creativeData?.news;
+
+  const { data: saveTheWorldData, isLoading: saveTheWorldLoading } =
+    useFetch<newsFromAPI>('/api/news/?mode=stw');
+  const saveTheWorldNews = saveTheWorldData?.news;
+
+  useEffect(() => {
+    window.addEventListener('scroll', () => {
+      if (window.pageYOffset > 10) {
+        setShowToTop(true);
+      } else {
+        setShowToTop(false);
+      }
+    });
+  }, []);
 
   return (
-    <section className={styles.news}>
-      <Header textThin="Fort" textBold="Knights" icon="fortnite" />
-      <main>
-        <h2>Battle Royale</h2>
-        <section className={styles.news__itemGroup}>
-          {battleRoyaleNews?.map((item) => (
-            <Card
-              key={item.id}
-              title={item.body}
-              image={item.image}
-              date={item.title}
-              gamemode="br"
-            />
-          ))}
-          <h2 className={styles.news__h2}>Save the World</h2>
-          {saveTheWorldNews?.map((item) => (
-            <Card
-              key={item.title}
-              title={item.body}
-              image={item.image}
-              date={item.title}
-              gamemode="stw"
-            />
-          ))}
-          <h2 className={styles.news__h2}>Creative</h2>
-          {creativeNews?.map((item) => (
-            <Card
-              key={item.id}
-              title={item.body}
-              image={item.image}
-              date={item.title}
-              gamemode="creative"
-            />
-          ))}
-        </section>
-        <div className={styles.news__toDo}>
-          <ToTop color="var(--clr-white)" />
-          <span>
-            This is everything :)
-            <br />
-            Come back later for new great stuff!
-          </span>
-        </div>
-      </main>
-      <Navigation active="news" />
-    </section>
+    <>
+      <section className={styles.news}>
+        <Header textThin="Fort" textBold="Knights" icon="fortnite" />
+        <main>
+          <section className={styles.news__itemGroup}>
+            <h2 className={styles.news__h2_first}>Battle Royale</h2>
+            {battleRoyaleNews?.map((item) => (
+              <Card
+                key={item.id}
+                title={item.body}
+                image={item.image}
+                date={item.title}
+                gamemode="br"
+              />
+            ))}
+            <h2 className={styles.news__h2}>Save the World</h2>
+            {saveTheWorldNews?.map((item) => (
+              <Card
+                key={item.title}
+                title={item.body}
+                image={item.image}
+                date={item.title}
+                gamemode="stw"
+              />
+            ))}
```

Comment:
```suggestion
            {saveTheWorldNews && saveTheWorldNews.map((item) => (
              <Card
                key={item.title}
                title={item.body}
                image={item.image}
                date={item.title}
                gamemode="stw"
              />
            ))}
```
This achieves the same thing but is more readable in my opinion.